etna/transforms/decomposition/stl.py

```diff
@@ -75,7 +75,7 @@ def __init__(
                     model_kwargs = {"trend": "add"}
             else:
                 raise ValueError(f"Not a valid option for model: {model}")
-        elif isinstance(model, TimeSeriesModel):
+        elif isinstance(model, type) and issubclass(model, TimeSeriesModel):
```

Comment:
Why isinstance(model, type)?

Reply:
`isinstance` checks whether the model is an object of type `type`. Without this condition, for the `model = 10` case, we get a `TypeError: issubclass() arg 1 must be a class` instead of `ValueError: Model should be a string or TimeSeriesModel`